index.html

```diff
@@ -69,5 +69,59 @@
         </div>
     </div>
 
+    <div class="container">
+        <section class="goods">
+            <ul class="goods__list">
+                <li class="goods__item">
+                    <a href="#" class="goods__item-link">
+                        <div class="goods__item-wrap goods__item-wrap--vr">
+                            <div class="goods__item-img goods__item-img--1"></div>
```

Comment:
why img-1 you should use --vr here, not img--1
also redundant tag, you can use it as background on your parent tag pseudoelement

something like that: goods__list>goods__item+goods__link>goods__img.goods__img--vr